src/Controls/tests/DeviceTests/Elements/Editor/EditorTests.iOS.cs

```diff
@@ -84,6 +84,52 @@ await CreateHandlerAndAddToWindow(contentPage, async () =>
 					Assert.Equal(expectedFontFamily, placeholderLabel?.Font?.FamilyName);
 				});
 			}
+
+			[Fact]
+			public async Task PlaceholderHorizontalTextAlignment()
```

Comment:
This test is failing on macOS:

`Assert.Equal() Failure: Values differ\nExpected: Left\nActual:   Natural`

Reply:
@jsuarezruiz , i have modified the code changes. Please let me know if you have any concerns.